css/style.css

```diff
@@ -351,8 +364,15 @@ div.ctr {
   margin-bottom: 10px;
 }
 
-@-moz-document url-prefix(https://mcmerge.paas.allizom.org/) {
+@-moz-document url-prefix(https://mcmerge.paas.allizom.org) {
```

Comment:
I think we should remove the use of `url-prefix`, since it doesn't work in other browsers. I don't think it's the end of the world if we see a deploy link when testing from the local filesystem (if anything it ensures we see the actual layout).